scalableminds/webknossos
# Clean up and unify config, add typing

I cleaned up the config as much as I could without actually changing any keys (which would require changes in the deployments).
 - Remove unused config fields
 - Combine blocks
 - Unify ordering in application.conf and standalone*.conf
 - Add type hints to ConfigReaders

It may be worth to change the config options keys themselves, creating a unified semantics and naming convention, and use Durations instead of integers with unclear time units. I’d move that to a later PR, though, as it requires operators to change how they override parameters for their individual setups.

### URL of deployed dev instance (used for testing):
- https://cleanupconf.webknossos.xyz

### Steps to test:
- CI should run through
- should still be able to trace, login/logout
- Standalone setup should also still run the same

### Issues:
- fixes #5173

------
